src/components/Score.js

```diff
@@ -10,10 +10,9 @@ const Score = React.createClass({
     };
   },
   saveScore() {
-    // TODO: validation - pk2
     const sessionId = this.props.params.sessionId;
     const data = JSON.stringify({
-      sessionId: this.state.sessionId,
+      sessionId: sessionId,
       scopes: this.state.scores
```

Comment:
@wolfgangcodes do you think it makes sense to move all sessionId handling to App component and do stuff like this ^ with an accessor?